src/datomish/sqlite_schema.cljc

```diff
@@ -0,0 +1,61 @@
+;; This Source Code Form is subject to the terms of the Mozilla Public
+;; License, v. 2.0. If a copy of the MPL was not distributed with this
+;; file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+(ns datomish.sqlite-schema
+  #?(:cljs
+     (:require-macros
+      [datomish.pair-chan :refer [go-pair <?]]
+      [cljs.core.async.macros :refer [go]]))
+  #?(:clj
+     (:require
+      [datomish.sqlite :as s]
+      [datomish.pair-chan :refer [go-pair <?]]
+      [clojure.core.async :refer [go <! >!]])
+     :cljs
+     (:require
+      [datomish.sqlite :as s]
+      [datomish.pair-chan]
+      [cljs.core.async :as a :refer [<! >!]])))
+
+(def current-version 1)
+
+(def v1-statements
+  ["CREATE TABLE datoms (e INTEGER NOT NULL, a TINYINT NOT NULL, v BLOB NOT NULL, tx INTEGER NOT NULL, index_avet TINYINT NOT NULL DEFAULT 0, index_vaet TINYINT NOT NULL DEFAULT 0)"
+   "CREATE TABLE transactions (e INTEGER NOT NULL, a TINYINT NOT NULL, v BLOB NOT NULL, tx INTEGER NOT NULL, added TINYINT NOT NULL DEFAULT 1)"
+   "CREATE INDEX eavt ON datoms (e, a)" ;; No v -- that's an opt-in index.
+   "CREATE INDEX aevt ON datoms (a, e)" ;; No v -- that's an opt-in index.
+   "CREATE INDEX avet ON datoms (a, v, e) WHERE index_avet = 1" ;; Opt-in index: only if a has :db/index true.
+   "CREATE INDEX vaet ON datoms (v, a, e) WHERE index_vaet = 1" ;; Opt-in index: only if a has :db/valueType :db.type/ref
+   "CREATE INDEX tx ON transactions (tx)"])
+
+(defn create-current-version
+  [db]
+  (go-pair
```

Comment:
I think some libraries (e.g., Android) call `onCreate` and friends in transactions. I don't know if ours does — if it doesn't, we should do it explicitly here.